slices.gemspec

```diff
@@ -26,7 +26,7 @@ Gem::Specification.new do |s|
   s.add_dependency 'redcarpet'         , '~> 3.0.0'
   s.add_dependency 'RedCloth'          , '~> 4.2.9'
   s.add_dependency 'stringex'          , '~> 1.4.0'
-  s.add_dependency 'will_paginate'     , '3.0.pre4'
+  s.add_dependency 'will_paginate_mongoid', '~> 2.0.1'
```

Comment:
Well my thinking is that hopefully most people are following semver, so minor version increases should be fine. If we pin the minor version, it means people will be more restricted in which versions of that gem they can use (until we update the gemspec again).